### PR TITLE
Update to v2.5.0

### PR DESCRIPTION
**Pull request by setu4993**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by setu4993** [comment]
@pmlandwehr : This PR is ready for review. Looking forward to your feedback!

**Comment by setu4993** [comment]
@conda-forge/help-python : This PR is ready for review.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,52 +1,47 @@
...
+    - fabric.testing
   commands:
     - fab --help
 
 about:
-  home: http://fabfile.org
-  license_file: LICENSE
-  license: BSD 2-Clause
+  home: "http://fabfile.org"
+  license: BSD
```
Comment (scopatz) [inline recipe/meta.yaml]: This should be the original `BSD-2-clause`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,52 +1,47 @@
...
   license_family: BSD
-  summary: 'Fabric is a simple, Pythonic tool for remote execution and deployment.'
+  license_file: LICENSE
+  summary: "High level SSH command execution"
```
Comment (scopatz) [inline recipe/meta.yaml]: I think the original summary was better :wink:

Comment (setu4993) [reply recipe/meta.yaml]: Agreed! My bad; that was unintentional as I built it from a new `skeleton` file.

**Comment by scopatz** [comment]
Thanks @setu4993!

**Comment by setu4993** [comment]
Thank you for taking the time to review, @scopatz! Very much appreciated!

**Pull request merged**